MainModule/Server/Commands/Moderators.lua

```diff
@@ -5314,11 +5314,13 @@ return function(Vargs, env)
 
 						new.Parent = part
 						new.Disabled = false
-						local ret = Remote.MakeGuiGet(v,"Notification",{
-							Title = "Flight";
-							Message = "You are now flying. Press E to toggle flight.";
-							Time = 10;
-						})
+						pcall(function()
```

Comment:
You should probably do something like this instead. (Its most likely yielding because its waiting for a response from the client?)
```lua
Routine(Remote.MakeGuiGet, v, "Notification",{
	Title = "Flight";
	Message = "You are now flying. Press E to toggle flight.";
	Time = 10;
})
```